app/docker-compose.yml

```diff
@@ -16,7 +16,6 @@ services:
     restart: unless-stopped
     ports:
       - 6545:6545
-    platform: linux/amd64
```

Comment:
Can you put this back? I don't think we should change the docker compose file in this PR.